amida-tech/saraswati-hedis-results-api
# Feature/sar 480 randomizer

# Related Tickets



- [SAR-480](https://jira.amida-tech.com/browse/SAR-480)

# Other Repos' PR(s) Intended to Work With This PR

- LINK_TO_THE_PR or "None."

# How Things Worked (or Didn't) Before This PR
The calculator is doing a fine job of producing data for each day. However, we need variation for demo and testing purposes. 

# How Things Work Now (And How to Test)
Now we have the randomizer. However we had to add a feature to turn off the calculations so it can run. 

1. In your `.env`, add `CALCULATION_ACTIVE=false`.
2. Go into Mongo and remove the entries for a few days back. With MongoDB Compass try searching with `{ date: {$gt: new Date('2022-04-11')} } ` to make your life easier. 
3. Run `node randomizer.js`. Wait for it to finish, it takes a moment to ensure population of the Mongo DB is complete.
4. Check Mongo to ensure that the new records are in there. You can search against the same dates.
5. Check the dashboard to ensure it appears. 
6. If you turn `CALCULATION_ACTIVE` to true again, it will flatten the latest results again. 



Note that 



# Readiness



1. [ ] This PR passes all automated tests.
8. [ ] This PR has no linting errors.
9. [ ] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.
